app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/HomeContract.kt

```diff
@@ -11,6 +11,7 @@ interface HomeContract {
     interface View {
         fun onGetGenresSuccess(genres: List<Genres>?)
         fun onGetMoviesNowPlayingSuccess(movies: List<Movie>?)
+        fun onGetMoviesUpcomingSuccess(movies: List<Movie>?)
```

Comment:
```suggestion
        fun onGetMoviesUpcomingSuccess(movies: List<Movie>)
```
tương tự với onGetMoviesNowPlayingSuccess